src/store/mutations.ts

```diff
@@ -16,8 +16,12 @@ const mutations: MutationTree<CalChartState> = {
   setShowTitle(state, title: string): void {
     state.show.title = title;
   },
-  addStuntSheet(state, stuntSheet: StuntSheet): void {
-    state.show.stuntSheets.push(stuntSheet);
+  addStuntSheet(state): void {
+    state.show.stuntSheets.push(
+      new StuntSheet({
+        title: `Stuntsheet ${state.show.stuntSheets.length + 1}`,
```

Comment:
Should the title be the stuntsheet's next ID? This might create duplicate titles. If you create two stuntsheets, delete the first, and add another, then you will have two stuntsheets with the same title of 'Stuntsheet 2'

Reply:
Good call-out! I think exposing the `id` may be weird if 100 or so stuntsheets have been created in the course of creating a show. So when the user adds a new stuntsheet the title becomes `Stuntsheet 101`.

I'm okay with duplicate titles... It doesn't cause code issues I don't think. In the example you provided, I'm hoping that the user would eventually re-name the titles!

In some old shows, there are some instances of duplicate titles... For example, see `ABD actual final.shw`. The first two stuntsheets have the same title. https://github.com/calband/calchart/blob/master/shows/ABD%20actual%20final.shw